Makefile

```diff
@@ -474,14 +476,30 @@ npm-update: node-check node_modules
 	npm install --package-lock
 
 .PHONY: js
-js: node-check $(JS_DEST)
+js: node-check fomantic $(JS_DEST)
 
 $(JS_DEST): node_modules $(JS_SOURCES)
 	npx eslint web_src/js webpack.config.js
 	npx webpack
 
+.PHONY: fomantic
+fomantic: $(FOMANTIC_DEST_DIR)
+node_modules/fomantic-ui/src: node-check node_modules
+
+# TODO(das7pad): replace with resolve.alias when building fomantic with webpack
+node_modules/fomantic-ui/src/theme.config: node_modules/fomantic-ui/src
+node_modules/fomantic-ui/src/theme.config: web_src/fomantic/theme.config.less
+	@diff=$$(diff '$<' '$@'); \
+	if [ -n "$$diff" ]; then \
+		cp -f $< $@; \
+	fi;
```

Comment:
@lunny Missing files are not subject to diffs and product 0 output lines on stdout, hence the broken build.

There is a `-N` option that treats missing files as empty.
We can work with the return code too and the target body can be simplified to `diff -qN $< $@ || cp -f $< $@` (`-q` suppresses the diff).

<details>
<summary> full diff </summary>
  
```diff
diff --git a/Makefile b/Makefile
index 56152c53f..7b2ce3e34 100644
--- a/Makefile
+++ b/Makefile
@@ -489,10 +489,7 @@ node_modules/fomantic-ui/src: node-check node_modules
 # TODO(das7pad): replace with resolve.alias when building fomantic with webpack
 node_modules/fomantic-ui/src/theme.config: node_modules/fomantic-ui/src
 node_modules/fomantic-ui/src/theme.config: web_src/fomantic/theme.config.less
-       @diff=$$(diff '$<' '$@'); \
-       if [ -n "$$diff" ]; then \
-               cp -f $< $@; \
-       fi;
+       diff -qN $< $@ || cp -f $< $@
 
 $(FOMANTIC_DEST_DIR): node_modules/fomantic-ui/src
 $(FOMANTIC_DEST_DIR): node_modules/fomantic-ui/src/theme.config
```
</details>

Reply:
@das7pad I have changed that to 
```makefile
ifeq (,$(wildcard $@))
	cp $< $@
endif
```

Reply:
Now it will not be updated upon changing the local file in `web_src`.

Let's step back and just use the timestamp based logic of make to determine the need for a new copy call. Just call `cp -f $< $@` in the target body.